include/taskolib/Sequence.h

```diff
@@ -489,8 +494,13 @@ class Sequence
     /// Empty if indentation is correct and complete, error message otherwise
     std::string indentation_error_;
 
+    /// Sequence label.
     std::string label_;
+
+    /// Collection of steps.
     std::vector<Step> steps_;
+
+    /// Flag to validate if the sequence is running.
```

Comment:
I guess "validate" is the wrong word. "find out" or "determine" maybe?

Reply:
Ok. I thought that "validate" has the same meaning as 'confirm' or 'approve'.  I change it to 'determine'.